Dockerfile

```diff
@@ -13,7 +13,8 @@ ARG PLUGIN_SMTP_MAILER="smtp-mailer-1.1.4.zip"
 # note, they still have to be activated
 # advanced custom fields (acf)
 FROM google/cloud-sdk:alpine as gcloud
-WORKDIR /
```

Comment:
My guess is that kaniko makes temporary backups for WORKDIRS and it fails here when trying to copy over /dev/zero.  But that's just my guess based on [this issuye](https://github.com/GoogleContainerTools/kaniko/issues/2033)